rt/src/test/java/io/fabric8/maven/rt/BaseBoosterIT.java

```diff
@@ -248,18 +296,30 @@ protected void addRedeploymentAnnotations(Repository testRepository, String rela
                 new ByteArrayInputStream(pomFragmentStr.getBytes()),
                 "UTF-8");
 
-        int nOpenShiftProfile = getOpenshiftProfileIndex(model);
+        int nOpenShiftProfile = getProfileIndexUsingFmp(model);
         model.getProfiles().get(nOpenShiftProfile).getBuild().getPluginsAsMap().get(fabric8MavenPluginKey).setConfiguration(configurationDom);
         writePomModelToFile(pomFile, model);
     }
 
-    protected int getOpenshiftProfileIndex(Model aPomModel) {
+    protected int getProfileIndexUsingFmp(Model aPomModel) {
```

Comment:
It would be better to add second parameter as `fabric8MavenPluginKey`. This kind of change saves lot of effort when comes to refactoring.
Lets caller decide what value it has to pass.